libs/telemetry/include/telemetry/measurement.hpp

```diff
@@ -0,0 +1,73 @@
+#pragma once
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include <iosfwd>
+#include <string>
+#include <unordered_map>
+
+namespace fetch {
+namespace telemetry {
+
+class Measurement
+{
+public:
+  using Labels = std::unordered_map<std::string, std::string>;
```

Comment:
Unordered map will make it complex (for client) for search for specific labels in the serialized stream using for example simple regexp, unless client does full JSON parsing.